### PR TITLE
Create 2019-10-17.md

### PR DESCRIPTION
**Pull request by TheCopperMind**

### CODE REVIEW

File: _minutes/2019-10-17.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿---
+date: 2019-10-07 6:00pm
```
Comment (penalosa) [inline _minutes/2019-10-17.md]:
```suggestion
date: 2019-10-17 6:00pm
```

File: _minutes/2019-10-17.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+**Present**:
+Kyle Cotton, Lars Thalian Morstad, Dee Yeum, Patrick Kage, Tony Duggan, Lorenzo Baldini, Samuel Macleod, Mohammed Javad, Qais Patankar, Lorenzo Baldini, Alexandra Purcarera, Yannik Nelson, Findlay Smith, Ryan Schuller, Rokas Gudavičius.
```
Comment (penalosa) [inline _minutes/2019-10-17.md]:
```suggestion
Kyle Cotton, Lars Thalian Morstad, Dee Yeum, Patrick Kage, Tony Duggan, Lorenzo Baldini, Samuel Macleod, Qais Patankar, Lorenzo Baldini, Alexandra Purcarera, Yannik Nelson, Findlay Smith, Ryan Schuller, Rokas Gudavičius.
```

File: _minutes/2019-10-17.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+**Agenda**:
+* Welcome new committee members! First year rep and graphic designer.
+* Infball committee need drive access.
+* Infball this year will probably have less subsidization. 
+*  Mathsoc and hoppers events are coming along nicely.
```
Comment (penalosa) [inline _minutes/2019-10-17.md]:
```suggestion
* Mathsoc and hoppers events are coming along nicely.
```

File: _minutes/2019-10-17.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Minutes need to be put on the website properly.
+* HTB is coming along nicely, sponsored have been contacted. 
+* Goldman Sachs are potential sponsors for CompSoc and HTB.
+* Merch needs to move forward. We need to decide how much CompSoc can subsidize.
+* G-research payment issue has been resolved.
+* Suggestion for website updates: SIGweb reviews code and the committee reviews minutes etc. Everything should be done by pull requests. 
+* Palantir event: Hoppers have backed out of the event.
+ * Trademarks for CompSoc Edinburgh/Hack the Burgh: setup costs, contesting, SigInt (?)
```
Comment (penalosa) [inline _minutes/2019-10-17.md]:
```suggestion
* Trademarks for CompSoc Edinburgh/Hack the Burgh: setup costs, contesting, SigInt (?)
```

File: _minutes/2019-10-17.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* Welcome new committee members! First year rep and graphic designer.
+* Infball committee need drive access.
+* Infball this year will probably have less subsidization. 
+*  Mathsoc and hoppers events are coming along nicely.
+* Minutes need to be put on the website properly.
+* HTB is coming along nicely, sponsored have been contacted. 
+* Goldman Sachs are potential sponsors for CompSoc and HTB.
+* Merch needs to move forward. We need to decide how much CompSoc can subsidize.
+* G-research payment issue has been resolved.
+* Suggestion for website updates: SIGweb reviews code and the committee reviews minutes etc. Everything should be done by pull requests.
```
Comment (qaisjp) [inline _minutes/2019-10-17.md]: You happy now @neanias?

**Comment by qaisjp** [comment]
lol @penalosa i am trying to get more committee to approve

**Comment by penalosa** [comment]
@qaisjp You mean my approval isn't enough? 😭

**Comment by qaisjp** [comment]
nah I think @findoslice suggested that everyone at the meeting should "approve" the minutes? funnily enough that's not on the minutes :D i'm not sure if it was actually brought forward as a motion though.

**Pull request merged**